templates/prometheus.yaml.erb

```diff
@@ -2,13 +2,15 @@
 <% global_config = scope.lookupvar('::prometheus::global_config') -%>
 <% rule_files = scope.lookupvar('::prometheus::rule_files') -%>
 <% scrape_configs = scope.lookupvar('::prometheus::scrape_configs') -%>
+<% remote_read_configs = scope.lookupvar('::prometheus::remote_read_configs') -%>
```

Comment:
what happens when we get an empty array here?

Reply:
it ends up in the config as `"remote_read": []` and prometheus just ignores it